### PR TITLE
Build/package/deploy the Mac SkyShell target

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

**Comment by abarth** [comment]
Rather than elaborate this buildbot, we should make the infra build bot do this work.  We're using it for everything else now.  It should be able to do this stuff as well.

**Comment by chinmaygarde** [comment]
Once I figure out how, I will add the same steps to the infra bots as well. Go/No-Go on this in the meantime?

File: travis/build_xcode_sdk.sh [hunk 1]
```diff
@@ -11,6 +11,7 @@ WORKSPACE=${TARGET_TEMP_DIR}/tmp/flutter_build_workspace
 DEPOT_WORKSPACE=${TARGET_TEMP_DIR}/tmp/flutter_depot_tools
 SERVICES_SDK_DIR=FlutterServicesSDK
 SERVICES_ARCHIVE=${SERVICES_SDK_DIR}.zip
+FLUTTER_MAC_ZIP=FlutterMac.zip
```
Comment (abarth) [inline travis/build_xcode_sdk.sh]: This should be called artifacts.zip

**Comment by abarth** [comment]
I don't think we should land this patch.  Instead, I think we should get the new infra to the point where it replaces this buildbot and then turn down this build bot.

**Pull request closed without merging**